server/utils/mail-template/mail-template.util.js

```diff
@@ -0,0 +1,118 @@
+                    <tbody>
+                      <tr>
+                        <td valign="top" align="center">
+                          <p style="font-family: 'Segoe UI','Roboto','Oxygen','Ubuntu','Cantarell','Fira Sans','Droid Sans','Helvetica Neue',sans-serif;color:#4E5C6E;line-height:20px;font-size:1.5rem;">AUTHOR'S
+                            HAVEN</p>
+                        </td>
+                      </tr>
+                    </tbody>
+                  </table>
+                  <!-- Logo end here -->
+                  <!-- Main CONTENT -->
+                  <table width="100%" cellpadding="0" cellspacing="0" border="0" bgcolor="#ffffff" style="border-radius: 4px; box-shadow: 0 2px 8px rgba(0,0,0,0.05);">
+                    <tbody>
+                      <tr>
+                        <td height="40"></td>
+                      </tr>
+                      <tr style="font-family: -apple-system,BlinkMacSystemFont,Segoe UI; color:#4E5C6E; font-size:14px; line-height:20px; margin-top:20px;">
+                        <td class="content" colspan="2" valign="top" align="center" style="padding-left:90px; padding-right:90px;">
+                          <table width="100%" cellpadding="0" cellspacing="0" border="0" bgcolor="#ffffff">
+                            <tbody>
+                              <tr>
+                                <td align="center" valign="bottom" colspan="2" cellpadding="3">
+                                  <img alt="Authors Haven Image" width="80" src="https://res.cloudinary.com/dsfgp1nzh/image/upload/v1554399400/envelope.jpg" />
+                                </td>
+                              </tr>
+                              <tr>
+                                <td height="30" &nbsp;=""></td>
+                              </tr>
+                              <tr>
+                                <td align="center"> <span style="color:#48545d;font-size:22px; line-height: 24px;">
+                                    ${emailTitle}
+                                  </span>
+                                </td>
+          
+                              </tr>
+                              
+                              <tr>
+                                <td align="center" style="margin:50px;"> <span style="color:#48545d; font-size:14px; line-height:20px;">
+                                    ${message}
+                                  </span>
+                                </td>
+                              </tr>
+                              <tr>
+                                <td height="20"></td>
+                              </tr>
+                              <tr>
+                                <td valign="top" width="48%" align="center"> <span>
+                                    <a href="${url}" style="display:block; padding:15px 25px; background-color:#40617B; color:#ffffff; border-radius:3px; text-decoration:none;">Visit Authors Haven</a>
```

Comment:
Since the application url is not changing constantly, this should not be a variable. Hardcode the production app url here and remove the url parameter

Reply:
Thanks. this is noted @paradisekelechi, and will be fixed